exporter/clickhouseexporter/exporter.go

```diff
@@ -28,30 +29,31 @@ import (
 )
 
 type clickhouseExporter struct {
-	client        *sql.DB
-	insertLogsSQL string
+	client          *sql.DB
+	insertLogsSQL   string
+	insertTracesSQL string
 
 	logger *zap.Logger
 	cfg    *Config
 }
 
-func newExporter(logger *zap.Logger, cfg *Config) (*clickhouseExporter, error) {
+func newExporter(logger *zap.Logger, cfg *Config, createLogsTable, createTracesTable bool) (
+	*clickhouseExporter, error) {
 	if err := cfg.Validate(); err != nil {
 		return nil, err
 	}
```

Comment:
This is not needed. Validate is called implicitly when receiver is created https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/moved_config.go#L77

Reply:
OK , i remove it